tests/scanner/audit/retention_rules_engine_test.py

```diff
@@ -851,6 +821,128 @@ def test_bucket_on_multi_project_has_vio(self):
             fake_bucket, 'bucket retention on multi projects')
         self.assertEqual(got_violations, expected_violations)
 
+    yaml_str_bigquery_retention_on_projects = """
```

Comment:
Also check that this gets parsed into the correct number of rules (1 for tables, 0 for buckets)